pkg/search/proxy/store/multi_cluster_cache.go

```diff
@@ -390,3 +408,90 @@ func (c *MultiClusterCache) clientForClusterFunc(cluster string) func() (dynamic
 		return c.newClientFunc(cluster)
 	}
 }
+
+func (c *MultiClusterCache) fillMissingClusterResourceVersion(ctx context.Context, mcv *multiClusterResourceVersion, clusters []string, gvr schema.GroupVersionResource) error {
+	errChan := make(chan error)
+	var lock sync.Mutex
+	var wg sync.WaitGroup
+
+	for _, cluster := range clusters {
+		if _, ok := mcv.rvs[cluster]; ok {
+			continue
+		}
+
+		wg.Add(1)
+		go func(cluster string) {
+			defer wg.Done()
+			klog.V(5).Infof("fillMissingClusterResourceVersion gvr=%v cluster=%v", gvr, cluster)
+			rv, err := c.getClusterResourceVersion(ctx, cluster, gvr)
+			if err != nil {
+				errChan <- err
+				return
+			}
+			if rv == "" {
+				return
+			}
```

Comment:
Just a question: when the rv is empty, will an error occur when the whole rv is displayed at the end because the rv of a cluster is empty?